modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -152,11 +151,15 @@ export default class GPUGridAggregator {
 
     positionsBuffer && positionsBuffer.delete();
     position64Buffer && position64Buffer.delete();
-    this.deleteResources(framebuffers);
-    this.deleteResources(maxMinFramebuffers);
-    this.deleteResources(minFramebuffers);
-    this.deleteResources(maxFramebuffers);
-    this.deleteResources(meanTextures);
+    this.deleteResources([
+      framebuffers,
```

Comment:
Can we track some of these with `resources`?

Reply:
Some of these are used in several places, so keeping separate objects make it much easy to read and less error prone.